sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -974,6 +974,28 @@ def get_cmis_media_lanes_mask(self, api, appl, lport, subport):
 
         return media_lanes_mask
 
+    def reset_app_code_and_reinit_all_lanes(self, api):
+        """
+	   Rest and DP init all the lanes
+        """
+        api.set_application(0xff, 0, 0)
+        api.set_datapath_deinit(0xff)
+        if not api.scs_apply_datapath_init(0xff):
+            return False
+        return True        
+
+    def is_cmis_app_code_reset_required(self, api, app_new):
```

Comment:
@AnoopKamath would you mind renaming to `is_appl_reconfigure_required()`

Reply:
Take care, Thanks